packages/tool/site/client/src/pages/dispensePage.tsx

```diff
@@ -213,4 +213,8 @@ function containsParacetamol(medicationRequest: fhir.MedicationRequest): boolean
   return medicationRequest.medicationCodeableConcept.coding[0].code === ("39720311000001101")
 }
 
+export function shouldSendCustomFhirRequest(dispenseFormValues: DispenseFormValues) {
```

Comment:
seems unnecessary to put this function here, might as well just check this condition inline in createDispenseNotification.ts